### DISCUSSION TITLE
Why isn't this SVG image showing?

### DISCUSSION THREAD

**Discussion by UsernameToSell** [wieslawsoltes/Svg.Skia#184] [Q&A]
I have this SVG-graphic:

<img src="https://github.com/wieslawsoltes/Svg.Skia/assets/144011416/a0ab84ec-2e04-4d87-96f4-53903dd990d0" width=100>

I have the following code:

```
<UserControl xmlns="https://github.com/avaloniaui"
			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
			 x:Class="AvaloniaExperiments2.Views.DebugSvgView"
			 >
	<Panel>
		<Button Width="100" Height="100">
			<Svg Path="/Assets/close.svg"/>
		</Button>
	</Panel>
</UserControl>
```

And this is the result:

<img width="164" alt="dysfunctional" src="https://github.com/wieslawsoltes/Svg.Skia/assets/144011416/455b5e55-4383-42a3-a053-d8a36e17ef81">

Obviously, it doesn't show my svg.
Why?

Thanks in advance.

**Comment by wieslawsoltes** [comment]
Did you include in avalonia resources?
